src/System.Net.Sockets/src/System/Net/Sockets/SafeSocketHandle.Unix.cs

```diff
@@ -144,6 +144,14 @@ internal bool IsNonBlocking
             }
         }
 
+        internal bool IsUnderlyingBlocking
+        {
+            get
+            {
+                return !_underlyingHandleNonBlocking;
+            }
+        }
```

Comment:
There's a lot of double-negatives going on here :)  Should we change the polarity of _underlyingHandleNonBlocking and rename it to underlyingHandleBlocking (and update the usage in SetHandleNonBlocking)?

Reply:
Fine with me. I originally named it  `IsUnderlyingHandleNonBlocking` but then I needed `!IsUnderlyingHandleNonBlocking` and that was hard for me to decode. Unless somebody objects, I would do the rename as separate change to split functional changes from naming cleanup.